frontend/src/components/regions/AddCardReviewModal/Presentation.jsx

```diff
@@ -107,7 +107,7 @@ export default function Presentation({
               <TextareaAutosize
                 className={classes.textArea}
                 aria-label="your comments"
-                rowsMin={5}
+                minRows={5}
```

Comment:
this seems to have changed the visuals of the modal? which I don't think was intentional, see image:
![image](https://user-images.githubusercontent.com/54069197/194094352-6580871b-caf4-46fb-a450-9d165b0120cf.png)